src/components/Link/IstioObjectLink.tsx

```diff
@@ -12,21 +12,11 @@ interface Props {
   query?: string;
 }
 
-export const GetIstioObjectUrl = (
-  name: string,
-  namespace: string,
-  type: string,
-  subType?: string,
-  query?: string
-): string => {
+export const GetIstioObjectUrl = (name: string, namespace: string, type: string, query?: string): string => {
   const istioType = IstioTypes[type];
   let to = '/namespaces/' + namespace + '/' + Paths.ISTIO;
 
-  if (!!subType) {
-    to = to + '/' + istioType + '/' + subType + '/' + name;
-  } else {
-    to = to + '/' + istioType.url + '/' + name;
-  }
+  to = to + '/' + istioType.url + '/' + name;
```

Comment:
minor, but now without the condition on subtype, the two lines above could be merged